### PR TITLE
shell: improved online documentation for txtsnd

### LINKED ISSUE #4997: shell_commands: txtsnd doc doesn't make it clear that it is a link-layer command
From `help`:

```
txtsnd               send raw data
```

What kind of raw data? 

(also might need fixing in several Tutorials).

### PR DESCRIPTION
**Pull request by OlegHahm**
fixes #4997.

### CODE REVIEW

**Comment by OlegHahm** [comment]
5000! :fireworks: :tada: :balloon:

**Comment by kaspar030** [comment]
Gratulations!!!!

**Comment by authmillenon** [comment]
Gratz! Have you checked tutorials and readmes too?

**Comment by OlegHahm** [comment]
I did a `vim -p $(git grep -l txtsnd)`

**Comment by authmillenon** [comment]
`git grep` in the RIOT wiki revealed

1. https://github.com/RIOT-OS/RIOT/wiki/Creating-your-first-RIOT-project
2. https://github.com/RIOT-OS/RIOT/wiki/Family%3A-native
3. https://github.com/RIOT-OS/RIOT/wiki/Virtual-riot-network

1. and 2. seem to be deprecated though (with 1. being replacable by http://doc.riot-os.org/creating-an-application.html, I think)

**Comment by OlegHahm** [comment]
Do I get an ACK here anyway? The wiki is out of scope for a PR.

File: examples/default/README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Running the `help` command on an iotlab-m3:
 2015-09-16 16:57:17,742 - INFO # lsm303dlhc_init      Initializes the lsm303dlhc sensor driver.
 2015-09-16 16:57:17,744 - INFO # lsm303dlhc_read      Prints data from the lsm303dlhc sensor.
 2015-09-16 16:57:17,746 - INFO # ifconfig             Configure network interfaces
-2015-09-16 16:57:17,747 - INFO # txtsnd               send raw data
+2015-09-16 16:57:17,747 - INFO # txtsnd               Sends a custom string over the link layer
```
Comment (Kijewski) [inline examples/default/README.md]: "Sends a custom string *as is* over the link layer"?

**Comment by OlegHahm** [comment]
addressed @Kijewski's comment

**Comment by DipSwitch** [comment]
ACK :p

**Comment by OlegHahm** [comment]
squashed

**Comment by cgundogan** [comment]
double green .. GO

**Comment by cgundogan** [comment]
Unrelated: But please update [1] or mark it as out-dated (or tell for which RIOT version it works) as soon as you have some time. A lot of people are reading this and then asking on the mailing list why something is not working (;

[1] http://www.codeproject.com/Articles/840499/RIOT-Tutorial

**Comment by OlegHahm** [comment]
Yes, this tutorial is horrible outdated. I will take care about it ASAP.

**Pull request merged**